guide/src/building_and_distribution.md

```diff
@@ -58,6 +58,17 @@ pyo3 = { version = "...", features = ["abi3"]}
 
 3. Ensure that the `.whl` is correctly marked as `abi3`. For projects using `setuptools`, this is accomplished by passing `--py-limited-api=cp3x` (where `x` is the minimum Python version supported by the wheel, e.g. `--py-limited-api=cp35` for Python 3.5) to `setup.py bdist_wheel`.
 
+### Minimum Python version for `abi3`
+
+We provide `abi3-py36`/`abi3-py37`/... features to set the minimum required Python version for abi3 wheel.
```

Comment:
Let's maybe add just a little bit more introduction here?

```suggestion
Because a single `abi3` wheel can be used with many different Python versions, PyO3 has feature flags `abi3-py36`, `abi3-py37`, `abi-py38` etc. to set the minimum required Python version for your `abi3` wheel.
```

Reply:
:+1: